### ISSUE TITLE
java.io.IOException: Unable to delete file - Loom 0.2.7-SNAPSHOT

### ISSUE THREAD

**Issue by ghost** [FabricMC/fabric-loom#194]
With Loom 0.2.7-SNAPSHOT, I get the error when I try to do `gradlew genSources`
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project '1.16'.
> java.io.IOException: Unable to delete file: C:\Users\***\Documents\1.16\.gradle\loom-cache\remapped_mods\LibGui-1.8.1+20w15a@net.fabricmc.yarn.20w16a.20w16a+build.9-v2.jar
```
However it works when I downgrade to Loom 0.2.6-SNAPSHOT? I cleared the .gradlew file in both my user folder and the project folder and restarted the PC before downgrading and that didn't work. No programs that could be using the file in question was running (I checked Task Manager) and there were no daemons were running.
On Discord, me (JithuJilladi#9249) and an user called Reoseah#7934 were having this problem and they gave me the idea of downgrading Loom.

**Comment by biom4st3r** [comment]
Same issue. I checked with Process Explorer and gradle is locking the file and not releasing it. Downgrading to 0.2.6 also worked for me.

**Comment by ghost** [comment]
This is fixed in the latest commit by @modmuss50. To fix this issue, upgrade to Loom 0.2.7 and do `gradle --stop` and `gradlew  --refresh-dependencies`.
